src/electron/views/device-connect-view/components/android-setup/prompt-choose-device-step.tsx

```diff
@@ -99,4 +104,19 @@ export class PromptChooseDeviceStep extends React.Component<
 
         return <AndroidSetupStepLayout {...layoutProps}></AndroidSetupStepLayout>;
     }
+
+    private getListItems(): any {
+        const devices: DeviceInfo[] = this.props.androidSetupStoreData.availableDevices;
+        const items = devices.map(m => ({ metadata: m }));
+        return items;
+    }
+
+    private selectItemZeroInList(items: any): void {
+        if (items.length > 0) {
+            this.selection.setChangeEvents(false, true);
```

Comment:
The setChangeEvents lines here are to prevent the selection's change handler from getting invoked during the ctor. I originally omitted these lines, but the got errors because the component hadn't yet been mounted. I'll see if I can refactor this a bit